src/test/java/com/unstoppabledomains/TestUtils.java

```diff
@@ -10,8 +10,8 @@
 
 public class TestUtils {
 
-    public static final String TESTING_CNS_PROVIDER_URL = "https://mainnet.infura.io/v3/e0c0cb9d12c440a29379df066de587e6";
-    public static final String TESTING_INFURA_CNS_PROJECT_ID = "e0c0cb9d12c440a29379df066de587e6";
+    public static final String TESTING_UNS_PROVIDER_URL = "https://rinkeby.infura.io/v3/e0c0cb9d12c440a29379df066de587e6";
```

Comment:
Good idea with migration of all tests to testnet. 

Could you add more domains with new TLD's into tests?